### ISSUE TITLE
Add SSZ container: `SyncCommittee`

### ISSUE THREAD

**Issue by MegaRedHand** [lambdaclass/lambda_ethereum_consensus#74]
Follow these instructions: https://github.com/lambdaclass/lambda_ethereum_consensus/blob/main/native/ssz_nif/README.md

#### `SyncCommittee`

```python
class SyncCommittee(Container):
    pubkeys: Vector[BLSPubkey, SYNC_COMMITTEE_SIZE]
    aggregate_pubkey: BLSPubkey
```

**Comment by MegaRedHand** [comment]
Sure!

**Comment by Godspower-Eze** [comment]
Yes, I have. I intend making a PR on thursday

**Comment by Godspower-Eze** [comment]
But, if it's urgent, then someone else can take it

**Comment by MegaRedHand** [comment]
cc @phklive

**Comment by phklive** [comment]
Yes i will need to also implement this SSZ container to be able to implement the BeaconState. Will also do it.

**Comment by phklive** [comment]
> But, if it's urgent, then someone else can take it

It's kinda urgent. 

I will take it please take another `Good first issue` thanks man

Always glad to see you on repos @Godspower-Eze 😉

**Comment by Godspower-Eze** [comment]
> > But, if it's urgent, then someone else can take it
> 
> It's kinda urgent.
> 
> I will take it please take another `Good first issue` thanks man
> 
> Always glad to see you on repos @Godspower-Eze 😉

Thanks bro for the kind words. I will pick any issue in a bit
